### PR TITLE
Documentation updates

### LINKED ISSUE #26: How to use the repository with apt-get
Hey there,
First of all thanks for providing this feature! I've installed this now on our NEXUS and uploaded the first custom build deb file. But now, I don't really know how I would add the repository to apt. My repository URL is: https://nexus.example.com/repository/apt-hosted/

Any help is kindly appreciated,
Cheers Daniel

### PR DESCRIPTION
**Pull request by DarthHater**
This adds in the shell of some user facing docs for the apt plugin

This pull request makes the following changes:
* APT User Doc
* Updates to README that are sort of various, gitter, link to the apt user doc, requirements

It relates to the following issue #s:
* Fixes #26

### CODE REVIEW

File: docs/APT_USER_DOCUMENTATION.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+You can set up an apt proxy repository to access a remote repository location.
+
+To proxy a apt repository, you simply create a new 'apt (proxy)' as documented in 
+[Repository Management](https://help.sonatype.com/display/NXRM3/Configuration#Configuration-RepositoryManagement) in
+details. Minimal configuration steps are:
+
+- Define 'Name'
+- Define URL for 'Remote storage' e.g. []()
+- Select a 'Blob store' for 'Storage'
```
Comment (j-s-3) [inline docs/APT_USER_DOCUMENTATION.md:34]: For apt you also need to specify a distribution.

File: docs/APT_USER_DOCUMENTATION.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+### Configuring AptGet
+
+To configure apt-get to use Nexus Repository as a Proxy for remote apt sites 
+
+TBD
```
Comment (j-s-3) [inline docs/APT_USER_DOCUMENTATION.md:40]: For testing we do the following

1. Make a backup of /etc/apt/sources.list (cp /etc/apt/sources.list /etc/apt/sources.backup.list)
2. Remove sources.list (rm /etc/apt/sources.list)
3. Create a new empty sources.list (touch /etc/apt/sources.list)
4. Edit the sources.list and add the following lines (changing the distribution xenial to be whatever your distribution is)

deb http://nxrm-ip:8081/repository/apt-proxy/ xenial main restricted
deb-src http://nxrm-ip:8081/repository/apt-proxy/ xenial main restricted

**Review by j-s-3 — APPROVED** [review]
👍 Looks good. There is obviously some apt specific stuff that needs adding here but I understand you are trying to scaffold it out so that others can contribute (I'll happily be a contributor 😄 )

**Comment by bhamail** [comment]
APT is now part of Nexus Repository Manager. Version 3.17.0 includes the APT plugin by default.
If this is still an issue if using 3.17.0 or later please file an issue at https://issues.sonatype.org/.
Links to the new source code location are in the top level README.md

**Pull request closed without merging**